be/src/formats/csv/csv_reader.h

```diff
@@ -67,9 +71,12 @@ class CSVReader {
     using Field = Slice;
     using Fields = std::vector<Field>;
 
-    CSVReader(char record_delimiter, string field_delimiter)
-            : _record_delimiter(record_delimiter),
+    CSVReader(string record_delimiter, string field_delimiter, size_t _record_delimiter_length,
+              size_t _field_delimiter_length)
+            : _record_delimiter(std::move(record_delimiter)),
               _field_delimiter(std::move(field_delimiter)),
+              _record_delimiter_length(_record_delimiter_length),
```

Comment:
What's the relationship between `_record_delimiter.size()` and `_record_delimiter_length`?

Reply:
it is equivalent, I think _record_delimiter_length have a better performance

Reply:
Don't guess about the performance unless you've tested it. And if it does have a better performance, which I would be surprised,  init `_record_delimiter_length` with `_field_delimiter.size()` instead of adding an extra argument in the constructor.